source/include/core_sntp_client.h

```diff
@@ -471,4 +490,43 @@ SntpStatus_t Sntp_Init( SntpContext_t * pContext,
 /* @[define_sntp_init] */
 
 
+/**
+ * @brief Sends a request for time from the currently configured server (in the
+ * context).
+ * If the user has provided an authentication interface, the client
+ * authentication code is appended to the request before sending over the network
+ * by calling the @ref SntpGenerateAuthCode_t function of the
+ * @ref SntpAuthenticationInterface_t.
+ *
+ * @note This function will ONLY send a request if there is a server available
+ * in the configured list that had not rejected an earlier request for time.
```

Comment:
Nit: sounds better if the second clause is present tense
```suggestion
 * @note This function will ONLY send a request if there is a server available
 * in the configured list that has not rejected an earlier request for time.
```